File: Melobit_player_java/app/src/main/java/melo_beat/models/SongInfo/Result.java
package melo_beat.models.SongInfo;

import java.util.List;

public class Result{
	private boolean hasVideo;
	private Image image;
	private String releaseDate;
	private Album album;
	private boolean localized;
	private String title;
	private int duration;
	private boolean copyrighted;
	private List<ArtistsItem> artists;
	private String id;
	private Audio audio;
	private String lyrics;
	private String downloadCount;

	public boolean isHasVideo(){
		return hasVideo;
	}

	public Image getImage(){
		return image;
	}

	public String getReleaseDate(){
		return releaseDate;
	}

	public Album getAlbum(){
		return album;
	}

	public boolean isLocalized(){
		return localized;
	}

	public String getTitle(){
		return title;
	}

	public int getDuration(){
		return duration;
	}

	public boolean isCopyrighted(){
		return copyrighted;
	}

	public List<ArtistsItem> getArtists(){
		return artists;
	}

	public String getId(){
		return id;
	}

	public Audio getAudio(){
		return audio;
	}

	public String getLyrics(){
		return lyrics;
	}

	public String getDownloadCount(){
		return downloadCount;
	}
}

File: tina/app/src/main/java/com/example/musicplayer/mvvm/model/top10songs/Result.kt
package com.example.musicplayer.mvvm.model.top10songs

data class Result(
    val album: Album,
    val artists: List<ArtistX>,
    val audio: Audio,
    val copyrighted: Boolean,
    val downloadCount: String,
    val duration: Int,
    val hasVideo: Boolean,
    val id: String,
    val image: ImageXXX,
    val localized: Boolean,
    val releaseDate: String,
    val title: String
)